packages/amplify-schema-validator/jest.config.js

```diff
@@ -3,6 +3,7 @@ module.exports = {
   collectCoverage: true,
   coverageProvider: 'v8',
   collectCoverageFrom: ['<rootDir>/src/**/*.{ts,js}'],
+  coveragePathIgnorePatterns: ['/__tests__/'],
```

Comment:
Does this rule obviate the need for all the individual entries in the `package.json`s?

Reply:
`packages/amplify-schema-validator` for some reason has the jest config in `jest.config.js` instead of `package.json`.

We will switch all of the packages to use `jest.config.js` when we create a `jest.config.base.js`.

`jest.config.base.js`
```
modules.exports = {
  collectCoverage: true,
  coverageProvider: 'v8',
  collectCoverageFrom: ['<rootDir>/src/**/*.{ts,js}'],
  coveragePathIgnorePatterns: ['/__tests__/'],
  ...
}
```


`packages/*/jest.config.js`
```
const baseConfig = require('../../jest.config.base.js');
module.exports = {
  ...baseConfig,
  'rootDir': './',
  // other config unique to specific package
}
```